### DISCUSSION TITLE
No funcionan los botones de las luces del avión

### DISCUSSION THREAD

**Discussion by Alexis241168** [flybywiresim/aircraft#5552] [General]
Hola después de la última versión de A320Nx, lo botones para las luces de aterrizaje, taxi, nav y beacon, no se pueden encender ni apagar

**Comment by hotshotp** [comment]
please update your aircraft to latest dev. this is fixed there.

**Comment by Alexis241168** [comment]
Tengo actualizada la última versión de desarrollo y el problema continúa. Saludos
